README.md

```diff
@@ -118,6 +119,21 @@ tilesets upload-source <username> <source_id> file-1.geojson file-4.geojson
 # Reading from a directory will not distinguish between GeoJSON files and non GeoJSON files. All source files will be run through our validator unless you pass the `--no-validation` flag.
 tilesets upload-source <username> <source_id> ./path/to/multiple/files/
 ```
+### upload-raster-source
+
+```shell
+tilesets upload-source-source <username> <source_id> <file>
+```
+
+Uploads Raster files to a source for tiling. Accepts GeoTIFF, NetCDF and GRIB formats right now. Can be used to add data to a source or to replace all of the data in a source with the `--replace` flag.
```

Comment:
I don't think there's a way to append raster data, so it seems like `--replace` is unneeded (raster source will always replace previous one)